### PR TITLE
Add help text to workflow invocation states

### PR DESCRIPTION
**Pull request by jmchilton**
Fill in missing help text for job states, dataset details, etc...

Screenshot 2024-02-28 at 6 00 23 PM

## How to test the changes?
(Select all options that apply)
- [x] Instructions for manual testing are as follows:
  1. View the help text.

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: client/src/components/Help/terms.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+unix:
+  commandLine: |
+    Galaxy typically runs a class of computed programs called "command-line applications". A command
```
Comment (mvdbeek) [inline client/src/components/Help/terms.yml]:
```suggestion
    Galaxy typically runs a class of software called "command-line applications". A command
```

File: client/src/components/Help/terms.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    line is essentially a string that describes what application should be run and what parameters
+    it should be ran with.
```
Comment (mvdbeek) [inline client/src/components/Help/terms.yml]:
```suggestion
    it should be run with.
```

File: client/src/components/Help/terms.yml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+    More information on command lines can be found on [Wikipedia](https://en.wikipedia.org/wiki/Command-line_interface).
+  exitCode: |
+    An exit code is a numeric representation of how an application execution has completed.
```
Comment (mvdbeek) [inline client/src/components/Help/terms.yml]:
```suggestion
    An exit code is a numeric representation of how an application execution has completed. An exit code of value 0 typically indicates successful execution of an application, while other values indicate a failure.
```

File: client/src/components/Help/terms.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    More information on standard output can be found on [Wikipedia](https://en.wikipedia.org/wiki/Standard_streams).
+  stderr: |
+    When an application is executed most of its error messages are typically printed to a stream of
+    data called the "standard error" (often abbreviated as stderr). Other reporting, diagnostic, and
+    logging messages are typically printed to an alternative stream called "standard output".
+
+    More information on standard error can be found on [Wikipedia](https://en.wikipedia.org/wiki/Standard_streams).
+  traceback: |
+    A "traceback" is also often called a "stack trace". This is typically used to describe what is
+    happening at various levels of an application at an instant in time and can sometimes used to
```
Comment (mvdbeek) [inline client/src/components/Help/terms.yml]:
```suggestion
    happening at various levels of an application at an instant in time and can sometimes be used to
```

**Review by mvdbeek — APPROVED** [review]
I love it, this is simple and amazing, just what we need and have been needing for years!

**Comment by mvdbeek** [comment]
Why did you remove the highlight ? We should definitely mention this in the release notes.

**Comment by jmchilton** [comment]
Seems like a silly small thing to highlight to me, but I will defer to you.

**Pull request merged**